client/src/Clusters/StandardCluster/standard_cluster.js

```diff
@@ -23,7 +23,7 @@ class StandardCluster extends Component {
 
         return (
             <div className="section">
-                <div className="section-header">
+                <div className="section-header" key={data.id}>
```

Comment:
We don't need to add the key here since nothing is being iterated over. the Key mainly if you're rendering multiple components from a list of things

`[].map(asdf => {})`